File: lab2/Lab_2_2.py
n = int(input("Enter an positive integer between(inclusive) 10 and 100: "))
fCount = 0
bCount = 0
fbCount = 0
while(10 > n or n > 100):
    n = int(input("Please enter an viable input: "))
    
for i in range (1, n+1):
    if(i%7 == 0):
        print(i ," is skipped")
        continue
    elif(i%3 == 0 and i%5 == 0):
        print("FizzBuzz")
        fbCount += 1
    elif(i%3 == 0):
        print("Fizz")
        fCount += 1
    elif(i%5 == 0):
        print("Buzz")
        bCount += 1
    else:
        print(i)

print("---Summary---")
print("Fizz count : " , fCount)
print("Buzz count : " , bCount)
print("FizzBuzz count : " , fbCount)

File: lab2/Lab_2_2.cpp
#include <iostream> 

using namespace std;

int main(){
    cout << "Enter an positive integer between(inclusive) 10 and 100: ";
    int x;
    cin >> x;
    int fCount = 0;
    int bCount = 0;
    int fbCount = 0;

    while(x < 10 || x > 100){
        cout << "Enter an positive integer between(inclusive) 10 and 100: ";
        int x;
        cin >> x;
    }
    
    for(int i = 1; i < x+1; i++){
        if(i%7 == 0){
            cout << i << " is skipped." << endl;
        }
        else if(i%3 == 0 && i%5 == 0){
            cout << "FizzBuzz" << endl;
            fbCount++;
        }
        else if(i%3 == 0){
            cout << "Fizz" << endl;
            fCount++;
        }
        else if(i%5 == 0){
            cout << "Buzz" << endl;
            bCount++;
        }
        else{
            cout << i << endl;
        }
    }

    cout << "--- Summary ---" << endl;
    cout << "Fizz Count: " << fCount << endl;
    cout << "Buzz Count: " << bCount << endl;
    cout << "FizzBuzz Count: " << fbCount << endl;

    return 0;


}